src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
```suggestion
        int index = random.nextInt(Colors.values().length);
```
lets make random global, create a class field

Reply:
Done.